evcouplings/fold/protocol.py

```diff
@@ -576,10 +578,103 @@ def _discrete_count(x):
     return outcfg
 
 
+def complex_dock(**kwargs):
+    """
+    Protocol:
+    Predict 3D structure from evolutionary couplings
+
+    Parameters
+    ----------
+    Mandatory kwargs arguments:
+        See list below in code where calling check_required
+
+    Returns
+    -------
+    outcfg : dict
+        Output configuration of the pipeline, including
+        the following fields:
+
+        * docking_restraints_files
+    """
+    check_required(
+        kwargs,
+        [
+            "prefix", "ec_file",
+            "segments", "dock_probability_cutoffs",
+            "dock_lowest_count", "dock_highest_count", "dock_increase",
+        ]
+    )
+
+    prefix = kwargs["prefix"]
+    outcfg = {}
+
+    # make sure output directory exists
+    create_prefix_folders(prefix)
+
```

Comment:
Hmm actually I think a `verify_resources` in front of accessing the ec_file wouldn't hurt to make sure the file is actually there